client/models.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+from django.core.exceptions import ValidationError
+from buy_together_app.models import User
+
+
+class Client(User):
+    area = models.CharField(max_length=32,
+                            validators=[MinLengthValidator(1)],
+                            help_text="area")
+
+    @staticmethod
+    def get_all_clients():
+        """Fetches list of all Clients.
+        Args:
+            None.
+        Returns:
+            returns a queryset containing all the Clients.
+        """
+        return Client.objects.all()
+
+    @staticmethod
+    def save_client(client):
+        """Saves new client 'safely'
+        Args:
+            client - the Client wanted to be saved.
+        Returns:
+            None.
+        raises:
+        ValidationError error: if fields input aren't valid.
+        """
+        try:
+            Client.full_clean(client)
+            client.save()
+        except ValidationError as error:
+            raise error
+
+    @staticmethod
+    def delete_client(client):
+        """Deletes client 'safely'
+        Args:
+            client - the client wanted to be deleted
+        Returns:
+            None.
+        raises:
+        ValueError error: if the client is not in DB.
+        """
+        try:
+            client.delete()
```

Comment:
```suggestion
            Client.object.get(user_name=client.user_name).delete()
```
that will raise an error

Reply:
other than that LGTM